### DISCUSSION TITLE
BM, BAPR and BAL aren't train prot systems

### DISCUSSION THREAD

**Discussion by besentv** [hiddewie/OpenRailwayMap-vector#166] [Tagging and rendering]
These shouldn't be shown here. Yes, there are documented tags for these, but they aren't train protection systems nevertheless, but rather a form of operation.

![image](https://github.com/user-attachments/assets/ba6f6ad4-fc35-4377-ac11-d80118733be7)

**Comment by hiddewie** [comment]
Some of the pink systems were indeed documented on the wiki but not tagged.

We can remove them. 

Maybe the wiki should also be updated then, especially because the list of train protection systems in https://wiki.openstreetmap.org/wiki/OpenRailwayMap/Tagging#Train_protection_systems is used in the proposal https://wiki.openstreetmap.org/wiki/Proposal:Railway:train_protection for the new tag `railway:train_protection`.

**Comment by hiddewie** [comment]
For example one rail line just within France (from Hagenau) is tagged with
```
railway:bapr = yes
railway:kvb = yes
```
(see https://overpass-turbo.eu/?w=%22railway%3Abapr%22%3D*+global&R). How would you propose to tag such a line?

Maybe the tagging legend could be split into "train protection systems" and "block control systems".

The SNCF data portal has "mode de cantonnement" (somewhat translated into *partitioning mode*) defined:

![image](https://github.com/user-attachments/assets/6e910e29-f111-4857-ba65-1499f41c410a)

See https://data.sncf.com/explore/dataset/mode-de-cantonnement-des-lignes/map/?location=12,48.83738,7.65713&basemap=jawg.transports

And separately the KVB equipped lines: https://data.sncf.com/explore/dataset/lignes-equipees-de-kvb/map/?location=12,48.85997,7.57782&basemap=jawg.transports

**Comment by hiddewie** [comment]
This will be cleaned up with https://github.com/hiddewie/OpenRailwayMap-vector/pull/164. 

I will then convert the issue to a discussion for further display / tagging discussion.
